service/history/taskAllocator.go

```diff
@@ -65,68 +63,54 @@ func newTaskAllocator(shard shard.Context) taskAllocator {
 }
 
 // verifyActiveTask, will return true if task activeness check is successful
-func (t *taskAllocatorImpl) verifyActiveTask(taskNamespaceID namespace.ID, task interface{}) (bool, error) {
+func (t *taskAllocatorImpl) verifyActiveTask(taskNamespaceID namespace.ID, task interface{}) bool {
 	t.locker.RLock()
 	defer t.locker.RUnlock()
 
 	namespaceEntry, err := t.namespaceRegistry.GetNamespaceByID(taskNamespaceID)
 	if err != nil {
-		// it is possible that the namespace is deleted
-		// we should treat that namespace as active
-		if _, ok := err.(*serviceerror.NotFound); !ok {
-			t.logger.Warn("Cannot find namespace", tag.WorkflowNamespaceID(taskNamespaceID.String()))
-			return false, err
-		}
-		t.logger.Warn("Cannot find namespace, default to process task.", tag.WorkflowNamespaceID(taskNamespaceID.String()), tag.Value(task))
-		return true, nil
+		return false
```

Comment:
what if the namespace exists but just not loaded into registry cache yet? this could happen during startup.

Reply:
For a newly created namespace, no task will be created/loaded for a shard before the namespace itself is loaded in to registry? Seems like it's the case, but just want to double check.

Reply:
I switched this back to `true` which means to continue to process the task.